app/deploy/gcp/variables.auto.tfvars

```diff
@@ -0,0 +1,36 @@
+# Queue ID should be a secret later, but for development is fine supplied in var/objects
+queue_id                      = "81e93644-4af2-11ef-a58f-676a7833797e"
+region                        = "us-central1"
+worker_image                  = "docker.io/michaelendsley/worker:0.3.1"
```

Comment:
Update this to the latest?

Reply:
Definitely. This was written before the publishing MR was merged, and I've been working off my own pushed worker version to incorporate some changes. This will not be in the final PR :+1: